compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -1022,7 +1022,8 @@ public BLangNode transform(ObjectFieldNode objFieldNode) {
         Optional<Node> doc = getDocumentationString(objFieldNode.metadata());
         simpleVar.markdownDocumentationAttachment = createMarkdownDocumentationAttachment(doc);
 
-        addRedonlyQualifier(objFieldNode.readonlyKeyword(), objFieldNode.typeName(), simpleVar);
+        // TODO: add changes for `final`
+//        addRedonlyQualifier(objFieldNode.readonlyKeyword(), objFieldNode.typeName(), simpleVar);
```

Comment:
Lets remove this, if its no longer needed.

Reply:
@SupunS PR is not yet ready to be reviewed. Waiting for PR #25784 to be merged to clean up old commits from that PR. Also have to do small refactoring. Will let you knw once ready. :)